src/main/java/com/github/games647/flexiblelogin/FlexibleLogin.java

```diff
@@ -24,19 +20,30 @@
 import org.spongepowered.api.event.game.state.GameInitializationEvent;
 import org.spongepowered.api.event.game.state.GamePreInitializationEvent;
 import org.spongepowered.api.plugin.Plugin;
-import org.spongepowered.api.service.command.CommandService;
 import org.spongepowered.api.plugin.PluginContainer;
+import org.spongepowered.api.service.command.CommandService;
 import org.spongepowered.api.service.config.DefaultConfig;
 import org.spongepowered.api.text.Texts;
 import org.spongepowered.api.util.command.args.GenericArguments;
 import org.spongepowered.api.util.command.spec.CommandSpec;
 
+import java.io.File;
+
+import ninja.leaping.configurate.commented.CommentedConfigurationNode;
+import ninja.leaping.configurate.loader.ConfigurationLoader;
+
 @Plugin(id = "flexiblelogin", name = "FlexibleLogin", version = "0.2.7")
 public class FlexibleLogin {
 
-    private final PluginContainer pluginContainer;
-    private final Logger logger;
-    private final Game game;
+    private static FlexibleLogin instance = new FlexibleLogin();
+
+    public static FlexibleLogin getInstance() {
```

Comment:
It's better to use a variable referencing to a main class. A static variable have to be nulled after the plugin is disabled. Otherwise it leads to memory leaks.